pages/user guides/discover_nmap.md

```diff
@@ -18,7 +18,7 @@ Nmap is capable of producing its output in an XML file. It allows you to inspect
 ## Using Nmap with Tidal Tools
 By leveraging the power of **Tidal Tools**, you can send the output generated by Nmap to your Tidal Migrations account.
 1. Install [Tidal Tools](https://guides.tidalmg.com/tidal-tools.html)
-2. [Login to Tidal Tools](https://guides.tidalmg.com/tidal-tools.html#login) with **Tidal login**
+2. Connect Tidal Tools and your Tidal Migrations account with [Tidal login](https://guides.tidalmg.com/tidal-tools.html#login).
```

Comment:
Connect Tidal Tools and your Tidal Migrations account with [`tidal login`](https://guides.tidalmg.com/tidal-tools.html#login).

WDYT?

Reply:
ahh, I see. `tidal login` as a command instead of **Tidal** as a 'brand'.
👍